src/components/scenes/CoinRankingScene.tsx

```diff
@@ -78,7 +80,7 @@ const CoinRankingComponent = (props: Props) => {
         coinRanking={coinRanking}
         percentChangeTimeFrame={percentChangeTimeFrame}
         assetSubText={assetSubText}
-        onPress={() => navigation.navigate('coinRankingDetails', { coinRankingData: coinRankingDatas[index] })}
+        onPress={navigateToDetails(index)}
```

Comment:
No point in using useHandler as you call the function instead of passing it as a prop